### PR TITLE
inspirehep: fix the image

### PR DESCRIPTION
**Pull request by drjova**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,13 +5,24 @@
...
 COPY . /opt/inspire
 
-# install dependencies
-RUN ./scripts/bootstrap
+# FIXME: We need to install ``inspirehep``
+RUN pip install -e .
+
+EXPOSE 8080
+
+CMD ["gunicorn", "-b", "0.0.0.0:8080", "--forwarded-allow-ips", "*", "inspirehep.wsgi:application"]
```
Comment (benjamin-bergia) [inline Dockerfile]:
```suggestion
ENTRYPOINT ["inspirehep"]
CMD ["shell"]
```

File: Dockerfile [hunk 2]
```diff
@@ -5,13 +5,24 @@
...
+    pip install -r requirements.txt
+
 COPY . /opt/inspire
 
-# install dependencies
-RUN ./scripts/bootstrap
+# FIXME: We need to install ``inspirehep``
+RUN pip install -e .
+
+EXPOSE 8080
```
Comment (benjamin-bergia) [inline Dockerfile]:
```suggestion
```

File: Dockerfile [hunk 3]
```diff
@@ -5,13 +5,24 @@
 # inspirehep is free software; you can redistribute it and/or modify it under
 # the terms of the MIT License; see LICENSE file for more details.
 
-FROM python:3.6
-
-RUN pip install poetry==0.12.16
+FROM python:3.6.7
 
+ENV PATH="/root/.poetry/bin:$PATH"
 WORKDIR /opt/inspire
```
Comment (benjamin-bergia) [inline Dockerfile]:
```suggestion
EXPOSE 8080
```

**Comment by drjova** [comment]
@benjamin-bergia updated

**Pull request merged**